MarvinLee_Portfolio_FEND/css/main.css

```diff
@@ -53,13 +52,12 @@ body, html {
 }
 
 /*Use flexbox for main*/
-main {
+.main-content {
   display: flex;
   flex-wrap: wrap;
 }
 
-.mainImage {
-  order: -1;
+.main-image-container {
   width: 100%;
 }
 

```

Comment:
Do not use camelCase when naming you html classes. Instead join words with -

Reply:
I thought I have to use camelCase when naming the html classes. Thank you for letting me know this tip.